linkd/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+// Copyright © 2021 The Radicle Link Contributors
+//
+// This file is part of radicle-link, distributed under the GPLv3 with Radicle
+// Linking Exception. For full terms see the included LICENSE file.
+
+pub mod args;
+
+mod cfg;
```

Comment:
bikeshed: we've used `config` or `Config` everywhere else, so I'd vote we stay consistent on the naming.

Reply:
Can you point me to an approved RFC where we agreed on this? :bike: :derelict_house:

Reply:
touché